opal/core/search/search_rule.py

```diff
@@ -109,7 +109,57 @@ def query(self, given_query):
             raise SearchException(err.format(given_query['queryType']))
 
 
+class EpisodeTeam(SearchRuleField):
+    ALL_OF = "All Of"
+    ANY_OF = "Any Of"
+
+    display_name = "Team"
+    description = "The team that looked after that episode of care"
```

Comment:
I don't think that an episode of care is the subject of the verb in this sentence...